hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -20,6 +20,7 @@ accounts.lastThrottleExempt=100
 accounts.nodeRewardAccount=801
 accounts.stakingRewardAccount=800
 accounts.storeOnDisk=true
+accounts.releaseAliasAfterDeletion=true
```

Comment:
Does this setting also need to be in `application.properties`?

Reply:
I'm not sure :confused:. 

In exactly which `application.properties` file the flag should be added as well because there are 11 of them? Every new feature flag I've been added till now has been defined in `bootstrap.properties` only. Some of the `application.properties` files are completely empty or have a few definitions. So I'm curious what's the main idea behind `application.properties` files, how they differ from the `bootstrap.properties`, and when they should be updated?

Reply:
application.properties is used by the modular code, while bootstrap.properties is used by the mono code. There should be a matching set of files for each location. @netopyr should be able to provide more guidance.

Reply:
Kelly is correct.
The point here being due to the complexity of changes we sometime opt for separate mono and modularized PRs
So far as the feature set and configs in this case match up within the milestone we're good and there won't be divergence.
So this PR can update `bootstrap.properties` only and I believe @lukelee-sl will handle the modular update that updates `application.properties`

Reply:
This PR addresses both mono and mod codebases so if we need to add the properties to `application.properties` we should do it. 
My understanding was that we don't need to add to the `application.properties` files because we've added them in the hedera-config module [here](https://github.com/hashgraph/hedera-services/pull/10701/files#diff-abd78ef65c6c7e4f1277e5d485f804d88173af0db888f7bcd13e2be51fa0d328R37). 
If we need to add them can you point us to the file that has to be updated please @poulok @netopyr @lukelee-sl